src/Config.cpp

```diff
@@ -254,11 +254,15 @@ ConverterPtr Config::NewFromString(const std::string& json,
     name = doc["name"].GetString();
   }
 
-  ConfigInternal* impl = (ConfigInternal*)internal;
-  if (configDirectory.back() == '/' || configDirectory.back() == '\\')
-    impl->configDirectory = configDirectory;
-  else
-    impl->configDirectory = configDirectory + '/';
+  if (!configDirectory.empty()) {
+    ConfigInternal* impl = (ConfigInternal*)internal;
+    if (configDirectory.back() == '/' || configDirectory.back() == '\\')
+      impl->configDirectory = configDirectory;
+    else
+      impl->configDirectory = configDirectory + '/';
+  } else {
+    impl->configDirectory.clear();
```

Comment:
error: ‘impl’ was not declared in this scope

Reply:
aah my bad sorry

Reply:
didnt cherry picked the right commit. Should be better now